### PR TITLE
Connect redux with react components

### PR DESCRIPTION
**Pull request by GOlivierNation**
Connect redux with react components

### CODE REVIEW

File: src/components/CategoryPage.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CategoryPage = () => (
+  <div>
+    <button type="button">CHECK STATUS</button>
+  </div>
```
Comment (ShahierNashaat) [inline src/components/CategoryPage.js]: - [ ] Kindly be noticed that the button `CHECK STATUS` should dispatch the `underConstruction` action creator.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @GOlivierNation , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/CategoryPage.js (hunk 1 above)
Comment (GOlivierNation) [reply src/components/CategoryPage.js]: Hi @ShahierNashaat I'm not getting this suggestion well, could you please make it clear?

Comment (ShahierNashaat) [reply src/components/CategoryPage.js]: It means when the `CHECK STATUS` button is clicked the action `bookStore/categories/CHECK_STATUS` should be dispatched.

Comment (GOlivierNation) [reply src/components/CategoryPage.js]: @ShahierNashaat  But it is working 

const checkReducer = (state = categoriesState, { type }) => {
  switch (type) {
    case CHECK_STATUS:
      return { categories: ['Under Construction...'] };
    default:
      return state;
  }
};

Comment (ShahierNashaat) [reply src/components/CategoryPage.js]: I know it is working but you should dispatch this action as you dispatch the create book action when the Add book button clicked.

Comment (GOlivierNation) [reply src/components/CategoryPage.js:6]: It's not fair as long as it is not in the project requirement.

Comment (ShahierNashaat) [reply src/components/CategoryPage.js]: Screenshot from 2022-06-15 22-17-47

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @GOlivierNation 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**